trio/_core/_tests/test_windows.py

```diff
@@ -216,7 +220,7 @@ async def test_too_late_to_cancel():
         assert target[:6] == b"test2\n"
 
 
-def test_lsp_that_hooks_select_gives_good_error(monkeypatch):
+def test_lsp_that_hooks_select_gives_good_error(monkeypatch) -> None:
```

Comment:
This one is [pytest.MonkeyPatch](https://docs.pytest.org/en/stable/reference/reference.html#pytest.MonkeyPatch).

Reply:
Changed in c06e788